.gitattributes

```diff
@@ -1,2 +1,5 @@
 model-manifest.json binary
 tests/js/test-balancer/jest-balance.json binary
+
```

Comment:
remove whitespace and sort -- also I'm not sure that svg files should be considered binary :thinking: